src/models/multiChart.js

```diff
@@ -213,6 +213,9 @@ nv.models.multiChart = function() {
                 return a.map(function(aVal,i){return {x: aVal.x, y: aVal.y + b[i].y}})
             }).concat([{x:0, y:0}]) : [];
 
+            x = lines1.xScale();
```

Comment:
yea, this doesn't seem right... this seems like it'll just throw away the:

    x   .domain(d3.extent(d3.merge(series1.concat(series2)), function(d) { return d.x }))
                 .range([0, availableWidth]);		                 .range([0, availableWidth]);

And even then, you're not changing the xScale of anything but lines1.   I think what you probably want to do is add a new option on multiChart named xScale and have it then control the xScale method on all subcharts (or at least the ones that have that option).